ambry-network/src/main/java/com/github/ambry/network/http2/Http2MultiplexedChannelPool.java

```diff
@@ -43,8 +43,10 @@
 import java.net.InetSocketAddress;
 import java.nio.channels.ClosedChannelException;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 import java.util.Properties;
+import java.util.Random;
```

Comment:
nit: these two and `VerifiableProperties` can be removed. (format this file)

Reply:
fixed. (for some reason, my cmd+L can't fix unused package anymore)